offline_cam/csv_iterator.hpp

```diff
@@ -0,0 +1,111 @@
+#include <fstream>
```

Comment:
maybe we need to have a single parsing for csv files instead one per folder

Reply:
Yes, it could work if we move the csv_iterator.hpp to the common folder. But since its usage is quite limited (only used by two plugins but not widespread across the system like others in common), it's also okay to keep it as it is now.